src/getStoreHandlers.js

```diff
@@ -1,5 +1,7 @@
 function format (string) {
+  /*eslint-disable no-unused-vars*/
   const [ _, action, state ] = `${ string }`.match(/(\w*)\_\d+\_(\w*)/, ' ') || []
+  /*eslint-enable no-unused-vars*/
```

Comment:
Yeah, there's some discussion around this here: https://github.com/eslint/eslint/issues/1939

Could probably write a custom rule for this.